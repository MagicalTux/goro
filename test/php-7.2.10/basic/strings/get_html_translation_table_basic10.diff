001- int(100)
002- Array
003- (
004-     ["] => &quot;
005-     [&] => &amp;
006-     [<] => &lt;
007-     [>] => &gt;
008-     [ ] => &nbsp;
009-     [¡] => &iexcl;
010-     [¢] => &cent;
011-     [£] => &pound;
012-     [¤] => &curren;
013-     [¥] => &yen;
014-     [¦] => &brvbar;
015-     [§] => &sect;
016-     [¨] => &uml;
017-     [©] => &copy;
018-     [ª] => &ordf;
019-     [«] => &laquo;
020-     [¬] => &not;
021-     [­] => &shy;
022-     [®] => &reg;
023-     [¯] => &macr;
024-     [°] => &deg;
025-     [±] => &plusmn;
026-     [²] => &sup2;
027-     [³] => &sup3;
028-     [´] => &acute;
029-     [µ] => &micro;
030-     [¶] => &para;
031-     [·] => &middot;
032-     [¸] => &cedil;
033-     [¹] => &sup1;
034-     [º] => &ordm;
035-     [»] => &raquo;
036-     [¼] => &frac14;
037-     [½] => &frac12;
038-     [¾] => &frac34;
039-     [¿] => &iquest;
040-     [À] => &Agrave;
041-     [Á] => &Aacute;
042-     [Â] => &Acirc;
043-     [Ã] => &Atilde;
044-     [Ä] => &Auml;
045-     [Å] => &Aring;
046-     [Æ] => &AElig;
047-     [Ç] => &Ccedil;
048-     [È] => &Egrave;
049-     [É] => &Eacute;
050-     [Ê] => &Ecirc;
051-     [Ë] => &Euml;
052-     [Ì] => &Igrave;
053-     [Í] => &Iacute;
054-     [Î] => &Icirc;
055-     [Ï] => &Iuml;
056-     [Ð] => &ETH;
057-     [Ñ] => &Ntilde;
058-     [Ò] => &Ograve;
059-     [Ó] => &Oacute;
060-     [Ô] => &Ocirc;
061-     [Õ] => &Otilde;
062-     [Ö] => &Ouml;
063-     [×] => &times;
064-     [Ø] => &Oslash;
065-     [Ù] => &Ugrave;
066-     [Ú] => &Uacute;
067-     [Û] => &Ucirc;
068-     [Ü] => &Uuml;
069-     [Ý] => &Yacute;
070-     [Þ] => &THORN;
071-     [ß] => &szlig;
072-     [à] => &agrave;
073-     [á] => &aacute;
074-     [â] => &acirc;
075-     [ã] => &atilde;
076-     [ä] => &auml;
077-     [å] => &aring;
078-     [æ] => &aelig;
079-     [ç] => &ccedil;
080-     [è] => &egrave;
081-     [é] => &eacute;
082-     [ê] => &ecirc;
083-     [ë] => &euml;
084-     [ì] => &igrave;
085-     [í] => &iacute;
086-     [î] => &icirc;
087-     [ï] => &iuml;
088-     [ð] => &eth;
089-     [ñ] => &ntilde;
090-     [ò] => &ograve;
091-     [ó] => &oacute;
092-     [ô] => &ocirc;
093-     [õ] => &otilde;
094-     [ö] => &ouml;
095-     [÷] => &divide;
096-     [ø] => &oslash;
097-     [ù] => &ugrave;
098-     [ú] => &uacute;
099-     [û] => &ucirc;
100-     [ü] => &uuml;
101-     [ý] => &yacute;
102-     [þ] => &thorn;
103-     [ÿ] => &yuml;
104- )
105- Done
001+ Fatal error: Uncaught Error: Call to undefined function get_html_translation_table in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/get_html_translation_table_basic10.php:6
002+ Stack trace:
003+ #0 {main}
004+   thrown in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/get_html_translation_table_basic10.php on line 6
