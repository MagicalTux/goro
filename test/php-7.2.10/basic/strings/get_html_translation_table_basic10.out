Fatal error: Uncaught Error: Call to undefined function get_html_translation_table in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/get_html_translation_table_basic10.php:6
Stack trace:
#0 {main}
  thrown in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/get_html_translation_table_basic10.php on line 6